time stamp              # actual time stamp of this measurement
string sample_token     # unique sample identifier for evaluation
bool is_key_frame       # whether the sample is annotated